### DISCUSSION TITLE
Symgony 7.2: Undefined variable $_v0

### DISCUSSION THREAD

**Discussion by bilik** [symfony/symfony#59049] [Q&A]
After the version update (7.1->7.2), an exception occurs in the twig above the code: {{ ('title')|trans }}

RuntimeError: An exception has been thrown during the rendering of a template ("Warning: Undefined variable $_v0").

This is a completely incomprehensible message. Does anyone have any idea what to do with it? Thanks

![obrazek](https://github.com/user-attachments/assets/142acb70-8b07-48ee-b0de-d262158d44b0)

**Comment by dimanche-matin** [comment]
- https://github.com/EasyCorp/EasyAdminBundle/issues/6605
- https://github.com/twigphp/Twig/issues/4480

**Comment by bilik** [comment]
Yes, this problem is described exactly in issue [#4480](https://github.com/twigphp/Twig/issues/4480). Although the issue is closed, it confirms the persistence of the bug, which is still not fixed. 

Downgrading the version twig/twig to 3.14.2 will solve the problem.

Moreover, it is interesting that searching for "Undefined variable $_v0" only finds my issue here, apparently there is a bug in the search function on github as well.
